Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:latest
```

Comment:
Not sure what this Dockerfile is for?
Is it different from files in the `docker` directory?

Reply:
This Dockerfile is for generating a Docker image (hosted on DockerHub) which will be used to create Sugarizer Server containers on the Kubernetes Cluster. It's different from the Dockerfile in the docker directory.
- It does not require MongoDB like [here](https://github.com/llaske/sugarizer-server/blob/master/docker-compose.yml).
- It separately clones Sugarizer and Sugarizer Server and installs Sugarizer Server dependencies.

Reply:
Okay, make sense. But to stay consistent, I suggest to move it in the `docker` directory and call it something like `Dockerfile-standalone` to make a difference with files `Dockerfile-mongodb` and `Dockerfile-server` already is this directory.

Reply:
@llaske I've moved it inside the `docker` directory.